test/client/storage.test.ts

```diff
@@ -37,4 +41,72 @@ describe("StateMemoryEngine", function() {
         engine.revert(checkpoint);
         assert.equal(engine.root, root1, "revert to previous root");
     });
+
+    describe("findVacantSubtree", () => {
+        it("finds first vacant subtree when empty", async function() {
+            const { path, witness } = await engine.findVacantSubtree(
+                maxSubtreeDepth
+            );
+            assert.equal(path, 0);
+            assert.lengthOf(witness, 30);
```

Comment:
Is checking the length of the witness sufficient for this test, or is there additional witness verification we should do here and in other test cases?

Reply:
Added `computeRoot` verification to these test cases.